src/components/Cards/PreviewCard/PreviewCard.module.css

```diff
@@ -53,18 +53,9 @@
 	display: flex;
 	align-items: center;
 	justify-content: center;
-
 	width: 398.33px;
-	height: 220px;
```

Comment:
We will need a set height for the preview card image, as the preview card is the same height for all image sizes [in the design](https://www.figma.com/file/0zFJaRNODss8lxwRygUh24/UI-Wilder-Market?node-id=5925%3A144039). If it doesn't have a set height, the preview card will snap to a larger height after loading an image as in the video below
https://user-images.githubusercontent.com/12437916/136883894-f0f27c60-4722-432e-895f-87cb19271eb8.mov

Basically for the preview card we just need an image container with set dimensions, then the image is contained within that.